migrations/2025-01-14-023145_media_filter/up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE local_user
+    ADD COLUMN hide_media boolean DEFAULT FALSE NOT NULL;
+
+CREATE INDEX idx_post_url_content_type ON post USING gin (url_content_type gin_trgm_ops);
```

Comment:
Oops, looks like you'll need to drop this index in `down.sql`, to fix failing CI.